crates/astria-bridge-withdrawer/src/bridge_withdrawer/ethereum/watcher.rs

```diff
@@ -192,15 +195,20 @@ impl Watcher {
         u128,
         u64,
     )> {
-        // wait for submitter to be ready
-        let SequencerStartupInfo {
+        let startup::Info {
             fee_asset,
-            next_batch_rollup_height,
-        } = self
-            .submitter_handle
-            .recv_startup_info()
-            .await
-            .wrap_err("failed to get sequencer startup info")?;
+            starting_rollup_height,
+            ..
+        } = select! {
+            () = self.shutdown_token.cancelled() => {
+                info!("watcher received shutdown signal while waiting for startup");
```

Comment:
So this will lead to confusing errors: either this is an error or not. IMO there is no need to write an info event if it's covered by the error anyway.